tests/ipam/framework.go

```diff
@@ -0,0 +1,34 @@
+/*
+ * This file is part of the KubeVirt project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Copyright 2024 Red Hat, Inc.
+ *
+ */
+
+package ipam
+
+import (
+	. "github.com/onsi/ginkgo/v2"
+
+	"kubevirt.io/kubevirt/tests/decorators"
+)
+
+func SIGDescribe(text string, args ...interface{}) bool {
+	return Describe("[sig-network][wg-ipam] "+text, decorators.SigNetwork, decorators.WgIPAM, args)
```

Comment:
why decorators.SigNetwork
we don't want network filter to choose this, 

network lanes will fail with this (kind + ovn is needed)
and anyhow it should be ortgonal please

Reply:
I was following `OWNERS` file but make sense to remove sig-network so it's not selected by jobs.

Reply:
yeah , I agree. IMHO, this should only have the wg info.

If anyone wants to know which sig owns this, they should check the owner_aliases.